src/components/Avatar/Avatar.vue

```diff
@@ -356,12 +356,17 @@ export default {
 			this.isMenuLoaded = false
 			this.loadAvatarUrl()
 		},
+		showUserStatus: {
```

Comment:
I think `showUserStatus` is a yes or no, so it doesnt need to be watched or reactive, just wrap the code in mount with it?

Reply:
The reason to make it reactive was that `showUserStatus` is a property, so (if I am not mistaken) the parent component could update its value at any time. If the check is done only when mounted, if `showUserStatus` was originally false and then changed to true no status would be fetched and thus shown.

Of course it would be strange if the parent component changed the property after the creation, but I used a watch for maximum compatibility.